src/components/form/Form.ts

```diff
@@ -18,32 +18,34 @@ export default class Form extends Block<IForm> {
     super('div', props)
   }
 
+  // public submit(e: Event) { }
+
   addEvents(): void {
     const { events = {} } = this.props
-    const inputs: HTMLInputElement = this.element?.querySelectorAll('input')
-    inputs.forEach((item: HTMLInputElement) => {
+    const inputs: any = this.element?.querySelectorAll('input')
```

Comment:
Тут не any, а NodeListOf<HTMLInputElement>
Можно написать так
```suggestion
    const inputs: any = this.element?.querySelectorAll<HTMLInputElement>('input')
```